modules/deleting-the-cluster-logging-operator-by-using-the-web-console.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+// * uninstalling/uninstalling-logging.adoc
+
+
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-07-05
+:_mod-docs-content-type: PROCEDURE
+
+[id="deleting-the-cluster-logging-operator-by-using-the-web-console_{context}"]
+= Deleting the {clo} by using the web console
+
+Delete the {clo} by using the {product-title} web console.
+
+.Prerequisites
+* You have administrator permissions.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators*.
```

Comment:
Have the 2 perspectives been merged into 1 for some versions of OCP (4.19)?  Maybe just say "In the web console" and let the user figure it out - it would be hard to conditionalize the statement

Reply:
good idea, done globally